tests/utils.spec.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @vitest-environment node
+ */
+
+import { describe, expect, it } from 'vitest'
+import { mergeDeep } from '../src/utils'
+
+describe('utils', () => {
+  it('should be possible to replace a primitive value with another', () => {
+    // ARRANGE
```

Comment:
nit: you can remove the `// ARRANGE`, `// ACT`, `// ASSERT` comments as we don't use these in other files.